lib/node_modules/@stdlib/blas/ext/base/dapxsum/manifest.json

```diff
@@ -35,6 +35,10 @@
 			],
 			"libpath": [],
 			"dependencies": [
+        "@stdlib/napi/export",
```

Comment:
This file has mixed TABS and spaces. The indentation should be 2-space.

This file is also missing separate build configurations for benchmarks and examples. See, e.g., https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/blas/ext/base/dapxsumors/manifest.json.

Reply:
Added the missing confs.
For the indentation thing, I've run the document formatting tool on the file and they all look 2-spaces indented. If still not the correct result, please let me know how to deal with it